stylus/cozy-ui/defaults.styl

```diff
@@ -102,6 +102,9 @@ body
     &.coz-btn-alert--success
         @extend $button-alert--success
 
+:global .coz-btn--upload-menu-item
```

Comment:
I just realised it is an all new button, separated from `.coz-btn`, so forget my suggestions.